core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -766,11 +766,27 @@ class WorkspaceService(protected val userInfo: UserInfo,
             }
           }
           workspaces.map { workspace =>
+
+            val maybeMcWorkspace : Option[WorkspaceDescription] =
+             workspace.workspaceType match {
+              case WorkspaceType.McWorkspace =>
+                Option(workspaceManagerDAO.getWorkspace(workspace.workspaceIdAsUUID, userInfo.accessToken))
+              case WorkspaceType.RawlsWorkspace => None
+            }
+
+            val cloudPlatform = maybeMcWorkspace match {
+                case None => Option(WorkspaceCloudPlatform.Gcp)
+                case Some(mcWorkspace) =>
+                  if (mcWorkspace.getAzureContext != null) Option(WorkspaceCloudPlatform.Azure)
+                  else if (mcWorkspace.getGcpContext != null) Option(WorkspaceCloudPlatform.Gcp)
+                  else throw new RawlsException(s"unexpected state, no cloud context found for workspace ${workspace.workspaceId}")
```

Comment:
I'm seeing a code coverage warning that the exception case isn't covered.

Should be pretty easy to add a test for it.